code/modules/projectiles/guns/energy/special.dm

```diff
@@ -424,7 +426,49 @@
 					"<span class='danger'>You hear a low pulsing roar!</span>"
 					)
 	is_charging = 1
-	sleep(60)
+	sleep(40)
+	is_charging = 0
+	if(!istype(user.get_active_hand(), src))
+		return
+	msg_admin_attack("[key_name_admin(user)] shot with \a [src.type] [key_name_admin(src)]'s target (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[src.x];Y=[src.y];Z=[src.z]'>JMP</a>)")

```

Comment:
A flaw with no_attack_log in general, probably better suited for a seperate bugfix PR.